app/segments/(team)/chooseTeam.js

```diff
@@ -116,13 +131,22 @@ function ChooseTeam() {
           >
             Your request to join the team has been received
           </Text>
-        ) : state === "invited" ? (
+        ) : state === "invitelist" ? (
           <View
             style={{
               alignItems: "center",
               justifyContent: "center",
             }}
           >
+            <Text
+              style={{
+                fontSize: 16,
+                textAlign: "center",
+                color: "gray",
+              }}
+            >
+              You have been invited to the team
+            </Text>
             <Button
```

Comment:
Shoud clicking this button when a player is invited remove them from the invite list? The list might get a little crowded if it doesn't. I tested this with "Roman Wilson" and upon joining the invite is still there
![image](https://github.com/user-attachments/assets/ff0dbf2f-8394-47a0-a87b-3d1bf1041af0)

Reply:
I've just added a feature where if a player joined using an invitation, the invite will be removed. Otherwise I don't quite understand your question. Clicking "Remove" should remove the invitation